### DISCUSSION TITLE
新創 Framework 攜手 Google，跨入 Chromebook 市場推模組化筆電

### DISCUSSION THREAD

**Discussion by dinosaur1009** [1101-GS4719-C/Python-Community-GS4719-C#718] [科技新知]
Framework 新款 Chromebook 的規格包括高解析度（2,256×1,504）、螢幕比例 3:2 的顯示器，舒適易打的 1.5mm 鍵程鍵盤，精密成型的鋁製外殼，還有厚度 15.85mm、重量 1.3 公斤的便攜性。其搭載最新第 12 代 Intel Core i5-1240P 處理器，可在執行繁重工作負載的同時輕鬆執行多工任務；並且採用 Google 的 Titan C 安全晶片，並在 Google 的 8 年承諾下，2030 年 6 月之前能夠持續接收系統自動更新，讓這款 Chromebook 保有快速運行和安全保護。此外，機身上還有隱私開關的實體設計，可以直接切斷鏡頭和麥克風的電源，保護用戶隱私。
內建的 ChromeOS 系統讓用戶可從 Google Play 下載的 Android App，使用 Crostini 在 Linux 上進行開發，並且能在 ChromeOS Alpha 版本透過線上遊戲平台 Steam 玩遊戲。
https://ccc.technews.tw/2022/09/22/introducing-the-framework-laptop-chromebook-edition/

雖然我還是.....會選擇mac（但mac環境安裝的時候問題真的很多）但感覺如果要換ㄉ話 也可以考慮（嗎

**Comment by Yunhans** [comment]
Chromebook的特點不是便宜嗎
這樣子會不會變很貴啊

**Reply by dinosaur1009** [reply]
剛剛看過....初步定價1000美金 比mac便宜一點點而已....

**Comment by williamyen0623** [comment]
太香了吧，看來google也準備在筆電走出自己的一條路了，非常期待Chromebook今後的表現！
